scripts/tei_transcriptions.py

```diff
@@ -0,0 +1,85 @@
+import glob
+import json
+import os.path
+
+import click
+from eulxml import xmlmap
+from eulxml.xmlmap import teimap
+from flask import current_app
+from flask.cli import with_appcontext
+
+
+class GenizaTeiLine(teimap.TeiLine):
+    name = xmlmap.StringField('local-name(.)')
+    lang = xmlmap.StringField('@xml:lang|tei:span/@xml:lang')
+
+
+class MainText(teimap.TeiDiv):
+    lines = xmlmap.NodeListField('tei:l|tei:label',
+                                 GenizaTeiLine)
+
+
+class GenizaTei(teimap.Tei):
+    # extend eulxml TEI to add mappings for the fields we care about
+    pgpid = xmlmap.IntegerField('tei:teiHeader//tei:idno[@type="PGP"]')
+    # normally main text content is under text/body/div; but at least one document has no div
+    text = xmlmap.NodeField('tei:text/tei:body/tei:div|tei:text/tei:body[not(tei:div)]', MainText)
+    lines = xmlmap.NodeListField('tei:text/tei:body/tei:div/tei:l',
+                                 GenizaTeiLine)
+    labels = xmlmap.NodeListField('tei:text/tei:body/tei:div/tei:label',
+                                  GenizaTeiLine)   # not really a line...
+
+
+@click.command()
+@with_appcontext
+def transcriptions():
```

Comment:
not super important, but i might give this a slightly more descriptive name, like `transcriptions_to_json()` or something

Reply:
The function name corresponds to the flask command name, so I'm inclined to leave it shorter (already kind of long). We can always rename when/if we have more transcription scripts.